### ISSUE TITLE
renderOnWindowResize is not work

### ISSUE THREAD

**Issue by baoduy** [planttheidea/remeasure#36]
I'm using the latest version of Chrome the component is not re-render when window size changed.

```javascript
 <Measured top width height renderOnWindowResize>
        {({ top }) => {
          const calHeight = window.innerHeight - top - 20;
          return <MyComponent   style={{ calHeight }}/>
        }}
</Measured>
```

**Comment by planttheidea** [comment]
I can verify that the component does indeed recalculate, but it is very likely that the computed values you are measuring (`height`, `top`, and `width`) have not changed, and therefore is not rerendering. That said ... the naming convention and the way I've documented the option, I can see how you would expect the component to _always_ rerender when the window is resized.

I'll work on that right now.

**Comment by baoduy** [comment]
@planttheidea you are right the Measured is not re-render as the top height and with of the component is not changed however, I would like the Measured re-render whenever the window size changes so that I can re-calculate the height of the component accordingtly.

That is greate feature I’m looking forward on next release.
Thanks for your help
Duy

**Comment by planttheidea** [comment]
Alright, I've just published `3.1.1` which should always rerender when `window.resize` is the trigger for the measuring event. I used your exact example as my test case, so it should give you what you seek.

Thanks for calling this out, if there is anything else let me know!

**Comment by baoduy** [comment]
Sure, thanks for your support.
